sdks/hermes-engine/utils/build-apple-framework.sh

```diff
@@ -16,15 +16,15 @@ REACT_NATIVE_PATH=${REACT_NATIVE_PATH:-$PWD/../..}
 JSI_PATH="$REACT_NATIVE_PATH/ReactCommon/jsi"
 
 function get_release_version {
-  ruby -rcocoapods-core -rjson -e "puts Pod::Specification.from_file('hermes-engine.podspec').version"
```

Comment:
Are we sure that in CI, where we invoke the `build-ios-framework` and the `build-mac-framework` we have the `RELEASE_VERSION`, `IOS_DEPLOYMENT_TARGET` and `MAC_DEPLOYMENT_TARGET` properly set? 
In [this case](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1070-L1080) we are not going through the `hermes-engine.podspec`, so it could be that these variables are not set...

Reply:
good catch, I guess we need to modify the CircleCI config to set them up too, right?

Reply:
thanks for following up the change. when i first proposed the change, i just want to keep it shorter. ideally we could add some if/else check.

for example,

```ruby
function get_release_version {
  if [[ -n "${RELEASE_VERSION}" ]]; then
    echo "${RELEASE_VERSION}"
  else
    ruby -rcocoapods-core -rjson -e "puts Pod::Specification.from_file('hermes-engine.podspec').deployment_target('ios')"
  fi
}
```

or

```ruby
function get_release_version {
  if [[ -z "${RELEASE_VERSION}" ]]; then
    echo "RELEASE_VERSION is not defined."
    exit 1
  fi
  echo "${RELEASE_VERSION}"
}
```

please feel free to change the code that makes sense to you.